.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Scala Merge CI
+
+on:
+  push:
+    branches: ['2.*.x']
+  pull_request:          ## TODO remove this line before merging
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build_and_test:
+    name: Windows
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - run: git config --global core.autocrlf false
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Note that we don't use olafurpg/setup-scala; it wouldn't buy us anything
+      # over setup-java. (We don't want csbt or xsbt; we prefer the standard
+      # sbt launcher, which comes preinstalled on Windows (and Ubuntu).
```

Comment:
Surprised to see unbalanced parens from someone with a Lisp background :stuck_out_tongue_winking_eye: